src/porepy/numerics/fracture_deformation/propagation_model.py

```diff
@@ -170,7 +170,7 @@ def _map_variables(self, x: np.ndarray) -> np.ndarray:
             # Only cell-based dofs have been considered so far.
             # It should not be difficult to handle other types of variables,
             # but the need has not been there.
-            dofs = self.equation_system._variable_dof_type[var]
```

Comment:
This usage reflects that the framework as it was and still is does not fully support changing numbers of dofs